OculusHeader hdr

uint32  deviceId
uint16  deviceType
uint16  partNumber
uint32  status

OculusVersionInfo versinInfo

uint32  ipAddr
uint32  ipMask
uint32  connectedIpAddr

uint8   macAddr0
uint8   macAddr1
uint8   macAddr2
uint8   macAddr3
uint8   macAddr4
uint8   macAddr5

float64 temperature0
float64 temperature1
float64 temperature2
float64 temperature3
float64 temperature4
float64 temperature5
float64 temperature6
float64 temperature7
float64 pressure
